config/jobs/kubernetes-sigs/cluster-api/cluster-api-periodics-release-1-2-upgrades.yaml

```diff
@@ -331,7 +331,54 @@ periodics:
     testgrid-alert-email: sig-cluster-lifecycle-cluster-api-alerts@kubernetes.io
     testgrid-num-failures-to-alert: "4"
 
-- name: periodic-cluster-api-e2e-workload-upgrade-1-25-latest-release-1-2
```

Comment:
@fabriziopandini  Q: This PR looks like we're planning to support 1.26 with release-1.2. I'm not sure if that's the plan (xref: https://github.com/kubernetes-sigs/cluster-api/issues/7671#issuecomment-1362728064)

(It even goes one step further by also testing v1.26 => v1.27 with CAPI v1.2.x)

Reply:
@aniruddha2000 I talked to Fabrizio and we would support v1.26 in v1.2 & v1.3. So you can ignore this comment.